### PR TITLE
update public  api documents

### PR DESCRIPTION
**Pull request by cdlliuy**
delete staled api docs and put the API design doc for temporary solution.

### CODE REVIEW

**Comment by cdlliuy** [comment]
@boyang9527  updated the commit with rst file , please review it again.

File: docs/Public_API.rst [hunk 1]
```diff
@@ -0,0 +1,948 @@
...
+| Name               | Description                                                                   | Valid values                                                        | Required              | Example values                   |
++--------------------+-------------------------------------------------------------------------------+---------------------------------------------------------------------+-----------------------+----------------------------------+
+| guid               | The GUID of the application                                                   |                                                                     | true                  |                                  |
++--------------------+-------------------------------------------------------------------------------+---------------------------------------------------------------------+-----------------------+----------------------------------+
+| start-time         | The start time                                                                | int, the number of nanoseconds elapsed since January 1, 1970 UTC.   | false                 | start-time=1494989539138350432   |
```
Comment (boyang9527) [inline docs/Public_API.rst]: add start time default when it is not specified, same for end time and other places

File: docs/Public_API.rst [hunk 2]
```diff
@@ -0,0 +1,948 @@
...
+
+cURL
+''''
+    | curl "https://[the-api-server-url]:[port]/v1/apps/8d0cee08-23ad-4813-a779-ad8118ea0b91/scaling\_histories?start-time=1494989539138350432&end-time=1494989539138399999&order=desc&page=1&results-per-page=10" \\
+    | -X GET \\
+    | -H "Authorization: bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoidWFhLWlkLTQwOCIsImVtYWlsIjoiZW1haWwtMzAzQHNvbWVkb21haW4uY29tIiwic2NvcGUiOlsiY2xvdWRfY29udHJvbGxlci5hZG1pbiJdLCJhdWQiOlsiY2xvdWRfY29udHJvbGxlciJdLCJleHAiOjE0NDU1NTc5NzF9.RMJZvSzCSxpj4jjZBmzbO7eoSfTAcIWVSHqFu5\_Iu\_o" 
+
+Response
+^^^^^^^^
+
```
Comment (boyang9527) [inline docs/Public_API.rst]: format the json body

**Comment by boyang9527** [comment]
Also please add description of the policy definition

File: docs/Public_API.rst (hunk 2 above)
Comment (cdlliuy) [reply docs/Public_API.rst]: fixed the json body

**Comment by cdlliuy** [comment]
@boyang9527 , the sample policy definition table is added. but not quite sure about the expression for a json array .  so just add scaling_rules part as an example.  
if everyone is fine with the format, I can go forward to add the definition for the more complex " schedules"

**Comment by cdlliuy** [comment]
@boyang9527 , updated the policy definition again.  please help to review

**Comment by cdlliuy** [comment]
@boyang9527 , updated the policy definition again. please help to review again  :-)

File: docs/Public_API.rst [hunk 3]
```diff
@@ -0,0 +1,985 @@
...
++--------------------------------------+------------------------+---------+----------------------------------------------------+
+| instance_max_count                   | int                    | true    |maximal number of instance count                    |
++--------------------------------------+------------------------+---------+----------------------------------------------------+
+| scaling_rules                        | Array<scaling_rules>   | true    |dynamic scaling rules                               |
++--------------------------------------+------------------------+---------+----------------------------------------------------+
+| schedules                            | Array<schedules>       | false   |schedule definition                                 |
++--------------------------------------+------------------------+---------+----------------------------------------------------+
+
+
+**Dynamic Scaling Rules Definition ``scaling_rules`` (part of the ``Policy`` configuration) :**
```
Comment (boyang9527) [inline docs/Public_API.rst]: why use ``

File: docs/Public_API.rst [hunk 4]
```diff
@@ -0,0 +1,985 @@
...
+| timezone                             | String                    | true    |Using timezone definition of Java.                               |
+|                                      |                           |         |https://docs.oracle.com/javase/8/docs/api/java/util/TimeZone.html|
++--------------------------------------+---------------------------+---------+-----------------------------------------------------------------+
+| recurring_schedule                   | Array<recurring_schedules>| false   |the schedules which will take effect repeatly                    |
++--------------------------------------+---------------------------+---------+-----------------------------------------------------------------+
+| specific_date                        | Array<specific_date>      | false   |the schedules which take effect only once                        |
++--------------------------------------+---------------------------+---------+-----------------------------------------------------------------+
+
+**Recurring Schedule Definition  ``recurring_schedule`` (part of the ``schedules`` configuration) :**
```
Comment (boyang9527) [inline docs/Public_API.rst]: same as above

**Comment by qibobo** [comment]
LGTM

**Pull request merged**